test/mock.js

```diff
@@ -224,12 +224,13 @@ function getAuthResp() {
 function setKustoQuery(functionName) {
   return {
     "query": `requests
-            | where operation_Name =~ '${functionName}'
-            | order by timestamp desc
-            | where success == "True" or success == "False"
-            | summarize errors = countif(success == "False"),invocations = countif(success == "True" or success == "False") by operation_Name
-            | extend details = pack_all()
-            | summarize Result = make_list(details)`, timespan: 'PT15M'
+    | where operation_Name =~ "${functionName}"
```

Comment:
let's change this query as well and pass `functionName` as array for function names.

Reply:
resolved